pkg/resources/sop_url.go

```diff
@@ -12,6 +12,7 @@ const (
 	sopUrlRedisResourceStatusPhaseFailed     = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/alerts/redis_resource_status_phase_failed.asciidoc"
 	sopUrlPostgresWillFill                   = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/alerts/postgres_storage_alerts.asciidoc"
 	sopUrlRedisMemoryUsageHigh               = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/alerts/redis_memory_usage_high.asciidoc"
+	sopUrlPostgresCpuUsageHigh               = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/alerts/postgres_cpu_usage_high.asciidoc"
```

Comment:
Just to confirm, this is planned to be merged? I'm seeing a 404

Reply:
Yep, pr is open for it here: https://github.com/RHCloudServices/integreatly-help/pull/652/files